ENIGMAsystem/SHELL/Bridges/xlib-OpenGL1/graphics_bridge.cpp

```diff
@@ -110,7 +110,7 @@ namespace enigma {
 }
 
 #include <Platforms/xlib/XLIBwindow.h> // window_set_caption
```

Comment:
change the <> to "" here

Reply:
It looks like @JoshDreamland actually did this: https://github.com/enigma-dev/enigma-dev/commit/5200d9f7fc018bcf556bfe89bd0a2f22201e583f#diff-40a9bc4254259eff578c582dad4aa23bL20

And he did it in the other bridges too. I wonder why?

Reply:
idk both will work but "" typically indicates our headers and <> is system ones

Reply:
Back in the day, I used `<>` wherever possible; `<>` meant search path, and `""` meant "this directory." I am fine with changing the meaning so that all/only ENIGMA files use `""`.

Reply:
I don't really like nitpicking about things unrelated to a pull request, but both of those already had a clearly defined meaning dictated by the standard, you didn't really need to invent your own meaning for them. Isn't it possible that `<>` will first look in the include path before looking elsewhere? Wouldn't that slow down the compile, even a little? If that is true then I will take up the habit of replacing `<>` with `""` where it is appropriate to do so.

Reply:
I decided I'll change it later. Those bridges need rewritten all around and we can address it then. I just want to get this PR merged sooner rather than later once Travis comes back online.

Reply:
Ok, I took care of this in 7b05efb1edd3d8c9d3e8ca3942472ccf953896dd which master already has from when I moved the window caption stuff.